clone代码之后，执行yarn install正常，再执行yarn examples:install报错

Build umd
×  error     Error: D:/source_code/qiankun/src/prefetch.ts(24,5): semantic error TS2717: Subsequent property declarations must have the same type.  Property 'requestIdleCallback' must be of type '(callback: IdleRequestCallback, options?: IdleRequestOptions | undefined) => number', but here has type '(callback: (deadline: RequestIdleCallbackDeadline) => void, opts?: RequestIdleCallbackOptions | undefined) => any'.

以上是报错信息。试过换npm命令，也是报相同的错。

--- a/src/prefetch.ts
+++ b/src/prefetch.ts
@@ -9,31 +9,10 @@ import { isFunction } from 'lodash';
 import { getAppStatus, getMountedApps, NOT_LOADED } from 'single-spa';
 import type { AppMetadata, PrefetchStrategy } from './interfaces';
 
-type RequestIdleCallbackHandle = any;
-type RequestIdleCallbackOptions = {
-  timeout: number;
-};
-type RequestIdleCallbackDeadline = {
-  readonly didTimeout: boolean;
-  timeRemaining: () => number;
-};
-
 declare global {
-  // eslint-disable-next-line @typescript-eslint/consistent-type-definitions
-  interface Window {
-    requestIdleCallback: (
-      callback: (deadline: RequestIdleCallbackDeadline) => void,
-      opts?: RequestIdleCallbackOptions,
-    ) => RequestIdleCallbackHandle;
-    cancelIdleCallback: (handle: RequestIdleCallbackHandle) => void;
-  }
-
-  interface Navigator {
-    connection: {
-      saveData: boolean;
-      effectiveType: string;
-      type: 'bluetooth' | 'cellular' | 'ethernet' | 'none' | 'wifi' | 'wimax' | 'other' | 'unknown';
-    };
+  interface NetworkInformation {
+    saveData: boolean;
+    effectiveType: string;
   }
 }
 
@@ -56,7 +35,7 @@ const isSlowNetwork = navigator.connection
   ? navigator.connection.saveData ||
     (navigator.connection.type !== 'wifi' &&
       navigator.connection.type !== 'ethernet' &&
-      /(2|3)g/.test(navigator.connection.effectiveType))
+      /([23])g/.test(navigator.connection.effectiveType))
   : false;
 
 /**

